.github/workflows/clang-tidy.yml

```diff
@@ -109,7 +109,7 @@ jobs:
           branch: "clang_tidy/autofix"
           branch-suffix: timestamp
           delete-branch: true
-          title: "[CI] clang-tidy auto fixes"
```

Comment:
Need to adjust `clang-tidy_on_fix_merged.yml` accordingly if changing this title 👍